dbms/src/Flash/Coprocessor/DAGExpressionAnalyzer.cpp

```diff
@@ -1216,6 +1224,8 @@ String DAGExpressionAnalyzer::appendCastIfNeeded(
     const ExpressionActionsPtr & actions,
     const String & expr_name)
 {
+    auto log = &Poco::Logger::get("LRUCache");
```

Comment:
why?
or use `Logger::get()`?

Reply:
why `LRUCache`?

Reply:
> why? or use `Logger::get()`?

Sorry, these are my debug codes and they shouldn't appear here